public/index.html

```diff
@@ -27,7 +27,14 @@ <h2 class="text-2xl text-green-50 italic">Frontend Developer-Writer</h2>
         <div class="absolute top-0 right-0 mt-1 mr-1">
             <img src="images/2202-2020-101115569136863779874-01 (1)-01-01.jpeg.jpg" alt="Adhithya Rajan's Photo" class="max-w-md rounded-full shadow-lg">
         </div>
-        <main class="bg-zinc-900 text-pink-700">
+        <main class="bg-zinc-900 text-pink-700 relative flex">
+            <div class="w-1/4 p-40">
+                <div class="mt-1 mr-1">
+                    <a href="https://docs.google.com/document/d/1S0JpUkxcW8MaNZTf6bv-dHpvvfPhD0kv/edit?usp=drivesdk&ouid=110576272185341512116&rtpof=true&sd=true">
```

Comment:
1. The link should open the resume in a new tab.
2. You can add your resume pdf to the source file and use that instead of the Google Drive link
  - create a `docs` or `assets` folder inside the public and place your resume pdf there & link to that
  - if you do this in the future when someone asks for your resume the link you share will be `adhithyarajan.com/resume` instead of some unreadable Google doc. (Note we need to purchase a domain for this)